app/core/services/realtime_stt.py

```
@@ -0,0 +1,316 @@
+                try:
+                    while True:
+                        chunk = await audio_q.get()
+                        if chunk is None:
+                            break
+                        if chunk:
+                            pcm = self._resample_to_24k(chunk, src_hz)
+                            await conn.input_audio_buffer.append(
+                                audio=base64.b64encode(pcm).decode()
+                            )
+                    await conn.input_audio_buffer.commit()
+                except Exception as exc:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)